src/rest/SearchEndpoint.ts

```diff
@@ -1019,6 +1019,13 @@ export class SearchEndpoint implements ISearchEndpoint {
       context[`context[${Utils.safeEncodeURIComponent(key)}]`] = value;
     });
 
+    if (queryObject.fieldsToInclude) {
```

Comment:
Would it be possible to break this block into a few more lines so it is easier to read. Maybe something like:
```
const fieldsToInclude = queryObject.fieldsToInclude.map(field => {
    const uri = Utils.safeEncodeURIComponent(field.replace('@', ''));
    return `"${uri}"`
})
queryParameters.fieldsToInclude = `[${fieldsToInclude.join(',')}]`
```